app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,25 @@
+class WebhooksController < ApplicationController
+  include ShopifyApp::WebhookVerification
+
+  class ShopifyApp::MissingWebhookJobError < StandardError; end
+
+  def receive
+    job_args = {shop_domain: shop_domain, webhook: webhook_params}
+    webhook_job_klass.perform_later(job_args)
```

Comment:
yeah! The best part is that with active job you can run it without a proper queue and add the queue for production. Basically it doesn't complicate development at all and its done proper.